solution/ui/regulations/alias.js

```diff
@@ -6,7 +6,7 @@ export const aliases = {
     "utilities": r('./utilities'),
     "legacy": r("../../regulations"),
     "sharedComponents": r("../eregs-component-lib/src/components/shared-components"),
-    "eregsComponentLib": r("../eregs-component-lib"),
+    "eregsComponentLib": r("../regulations/eregs-component-lib"),
```

Comment:
Yeah.  Its using the same settings in vite but vitest wont find the component otherwise.  Not sure exactly why but I didnt see any issues after the implementation